pkg/controller/backingstore/backingstore_controller.go

```diff
@@ -71,6 +71,10 @@ func Add(mgr manager.Manager) error {
 	if err != nil {
 		return err
 	}
+	err = c.Watch(&source.Kind{Type: &corev1.Secret{}}, ownerHandler, &filterForOwnerPredicate, &logEventsPredicate)
```

Comment:
this will only watch on secrets their owner is the backingstore, what about other secrets?

Reply:
The goal is to watch the backing store secrets, and not trigger a reconcile when not needed.

Reply:
of course you don't want to trigger reconcile when it's not needed, but you can create a backingstore while specifying a secret - then the backingstore won't own the secret and then you won't get triggers on these secrets at all

Reply:
Didn’t fully understood what you meant, can you explain again please?

Reply:
@kfir-payne look at how we reconcile bucket class on backingstore changes using `EnqueueRequestsFromMapFunc` in `bucketclass_controller`